compilerlibs/Makefile.compilerlibs

```diff
@@ -0,0 +1,331 @@
+# that they are consistent with the interface digests in the archives.
+
+UTILS=utils/config.cmo utils/build_path_prefix_map.cmo utils/misc.cmo \
+	utils/identifiable.cmo utils/numbers.cmo utils/arg_helper.cmo \
+	utils/clflags.cmo utils/profile.cmo utils/load_path.cmo \
+	utils/terminfo.cmo utils/ccomp.cmo utils/warnings.cmo \
+	utils/consistbl.cmo utils/strongly_connected_components.cmo \
+	utils/targetint.cmo utils/int_replace_polymorphic_compare.cmo \
+	utils/domainstate.cmo
+UTILS_CMI=
+
+PARSING=parsing/location.cmo parsing/longident.cmo \
+  parsing/docstrings.cmo parsing/syntaxerr.cmo \
+  parsing/ast_helper.cmo \
+  parsing/pprintast.cmo \
+  parsing/camlinternalMenhirLib.cmo parsing/parser.cmo \
+  parsing/lexer.cmo parsing/parse.cmo parsing/printast.cmo \
+  parsing/ast_mapper.cmo parsing/ast_iterator.cmo parsing/attr_helper.cmo \
+  parsing/builtin_attributes.cmo parsing/ast_invariants.cmo parsing/depend.cmo
+PARSING_CMI=\
+  parsing/asttypes.cmi \
+  parsing/parsetree.cmi
+
+TYPING=typing/ident.cmo typing/path.cmo \
+  typing/primitive.cmo typing/type_immediacy.cmo typing/types.cmo \
+  typing/btype.cmo typing/oprint.cmo \
+  typing/subst.cmo typing/predef.cmo \
+  typing/datarepr.cmo file_formats/cmi_format.cmo \
+  typing/persistent_env.cmo typing/env.cmo \
+  typing/typedtree.cmo typing/printtyped.cmo typing/ctype.cmo \
+  typing/printtyp.cmo typing/includeclass.cmo \
+  typing/mtype.cmo typing/envaux.cmo typing/includecore.cmo \
+  typing/tast_iterator.cmo typing/tast_mapper.cmo \
+  file_formats/cmt_format.cmo typing/untypeast.cmo \
+  typing/includemod.cmo typing/typetexp.cmo typing/printpat.cmo \
+  typing/parmatch.cmo typing/stypes.cmo \
+  typing/typedecl_properties.cmo typing/typedecl_variance.cmo \
+  typing/typedecl_unboxed.cmo typing/typedecl_immediacy.cmo \
+  typing/typedecl.cmo typing/typeopt.cmo \
+  typing/rec_check.cmo typing/typecore.cmo typing/typeclass.cmo \
+  typing/typemod.cmo
+TYPING_CMI=\
+  typing/annot.cmi \
+  typing/outcometree.cmi
+
+LAMBDA=lambda/debuginfo.cmo \
+  lambda/lambda.cmo lambda/printlambda.cmo \
+  lambda/switch.cmo lambda/matching.cmo \
+  lambda/translobj.cmo lambda/translattribute.cmo \
+  lambda/translprim.cmo lambda/translcore.cmo \
+  lambda/translclass.cmo lambda/translmod.cmo \
+  lambda/simplif.cmo lambda/runtimedef.cmo
+LAMBDA_CMI=
+
+COMP=\
+  bytecomp/meta.cmo bytecomp/opcodes.cmo \
+  bytecomp/bytesections.cmo bytecomp/dll.cmo \
+  bytecomp/symtable.cmo \
+  driver/pparse.cmo driver/compenv.cmo \
+  driver/main_args.cmo driver/compmisc.cmo \
+  driver/makedepend.cmo \
+  driver/compile_common.cmo
+COMP_CMI=\
+  file_formats/cmo_format.cmi \
+  file_formats/cmx_format.cmi \
+  file_formats/cmxs_format.cmi
+# cmxs_format is required for native compilation (which would suggest
+# a placement in ASMCOMP), but it is also a dependency of ocamlobjinfo
+# that we want to be able to build on bytecode-only systems if
+# possible.
```

Comment:
Technically, this is true also of `cmx_format`. Given that we want `ocamlobjinfo` to know about as many ocaml files as possible, I would rephrase it to "All file format descriptions are in the ocamlcommon library so that ocamlobjinfo can depend on them" or something similar. Also, I was wondering why `cmi_format` and `cmt_format` were missing, but since they have implementations they're actually included earlier.

Reply:
I rephrased following the suggestion, thanks.